packages/internal/src/storage-test-utils.js

```diff
@@ -30,7 +30,7 @@ export const makeFakeStorageKit = (rootPath, rootOptions) => {
       case 'set':
         for (const [key, value] of message.args) {
           if (value !== undefined) {
-            data.set(key, value);
+            data.set(key, [value]);
```

Comment:
That's sort of a breaking change. I guess it's an internal API, but I think I'll have to fix a pending PR to deal.

Reply:
i think I don't have a good handle on this api. not a big deal in any case.